test/swift-indent/basic.swift

```diff
@@ -83,6 +83,27 @@ bax(34949494949)
     .baz
 
 
+// Enum element parameters should be aligned, and raw values should be indented.
+
+enum TestEnum {
+    case first(x: Int,
+               y: Int,
+               z: Int),
+         second(
+            x: Int,
```

Comment:
3 columns?

Reply:
12 total. Lines that aren't column aligned themselves round down to the nearest tab-width. Not sure that's the best behavior, but its not new.

Reply:
I see, thanks.